models/course.js

```diff
@@ -19,11 +20,16 @@ const schema = new mongoose.Schema({
     },
   },
 })
-module.exports = mongoose.model('Course', schema)
+
```

Comment:
I don't get the point why you write the same code I've written before in a different style  ?